tests/unit/models/cart-model-test.js

```diff
@@ -73,20 +73,57 @@ test('it proxies sub total from the underlying cart', function (assert) {
   assert.equal(model.subtotal, cartFixture.cart.subtotal_price);
 });
 
-test('it creates a line item when you add a variant', function (assert) {
+test('it deprecates addVariants to createLineItemsFromVariants', function (assert) {
```

Comment:
Honestly, instead of duplicating the test, I would stub `createLineItemsFromVariants` and test that it's called by the deprecated method. Right now, this doesn't test that. It just tests that they have similar functionality (assuming the test bodies are the same).